src/test/java/org/testng/xml/XmlSuiteTest.java

```diff
@@ -11,15 +11,17 @@
 import java.util.Collections;
 import java.util.List;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 public class XmlSuiteTest extends SimpleBaseTest {
 
     @Test
     public void testIncludedAndExcludedGroups() {
         XmlSuite suite = new XmlSuite();
         suite.addIncludedGroup("foo");
         suite.addExcludedGroup("bar");
-        assertEquals(Collections.singletonList("foo"), suite.getIncludedGroups());
-        assertEquals(Collections.singletonList("bar"), suite.getExcludedGroups());
+        assertThat(suite.getIncludedGroups()).hasSameElementsAs(Collections.singletonList("foo"));
```

Comment:
Fixed.

Reply:
You can avoid the singleton list because assertj provides methods the check on elements